README.md

```diff
@@ -79,6 +79,12 @@ Simple test that checks whether a file or directory exists. This indicates it in
 ### get(path:  string):  Promise<string  |  Buffer>
 Gets a file as a string/Buffer.
 
+### getFileOrFolderCreator(path: string):  Promise\<string\>
```

Comment:
`getPathCreator`?

Reply:
Thanks :) I was looking for a better name.